src/seedu/addressbook/data/person/Address.java

```diff
@@ -8,12 +8,16 @@
  */
 public class Address {
 
-    public static final String EXAMPLE = "123, some street";
-    public static final String MESSAGE_ADDRESS_CONSTRAINTS = "Person addresses can be in any format";
-    public static final String ADDRESS_VALIDATION_REGEX = ".+";
+    public static final String EXAMPLE = "123, some street, some unit, some postalcode";
+    public static final String MESSAGE_ADDRESS_CONSTRAINTS = "Person addresses can only be in the format a/BLOCK, STREET, UNIT, POSTAL_CODE";
```

Comment:
Avoid lines longer than 110 chars. When a line is too long, it requires horizontal scrolling, that slows down the reader/reviewer.